### ISSUE TITLE
`not not True` is not detected as `WPS330`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#905]
# Bug report

## What's wrong

This code does not raise `WPS330`, but it should:

```python
not not True
```
